shell/platform/darwin/ios/framework/Source/FlutterAppDelegate.mm

```diff
@@ -210,6 +210,32 @@ - (void)application:(UIApplication*)application
   }
 }
 
+- (void)application:(UIApplication*)application
+    handleEventsForBackgroundURLSession:(nonnull NSString *)identifier
+                completionHandler:(nonnull void (^)())completionHandler {
+  for (id<FlutterPlugin> plugin in _pluginDelegates) {
+    if ([plugin respondsToSelector:_cmd]) {
```

Comment:
Nit: Instead of nested if blocks, prefer early `continue` or the a single statement with `&&`.

Reply:
I can make that change, but it looks like this pattern appears throughout FlutterAppDelegate. Is this something that should be addressed in another PR?

Reply:
nvm, I didn't read the rest of the file. Thanks.